java/clients/src/test/java/sleeper/clients/status/update/DeleteTableIT.java

```diff
@@ -119,12 +119,6 @@ void shouldDeleteOnlyTable() throws Exception {
         List<String> tableFilesInS3 = streamTableObjects(table)
                 .map(S3ObjectSummary::getKey)
                 .collect(Collectors.toList());
-        assertThat(tableFilesInS3.stream()
-                .filter(key -> key.startsWith(tablePrefix(table, "statestore/files"))))
-                .hasSize(2);
-        assertThat(tableFilesInS3.stream()
-                .filter(key -> key.startsWith(tablePrefix(table, "statestore/partitions"))))
-                .hasSize(1);
         assertThat(tableFilesInS3.stream()
                 .filter(key -> key.startsWith(tablePrefix(table, "data/partition_root")))
```

Comment:
Can we assert on all the files under the base table prefix at once now, rather than just looking under this directory?